Code/Tools/ProjectManager/Source/ProjectTemplateInfo.h

```diff
@@ -27,6 +27,8 @@ namespace O3DE::ProjectManager
         QString m_name;
         QString m_path;
         QString m_summary;
+        QString m_requirements;
+        QString m_license;
```

Comment:
you'll want to move this to the base `TemplateInfo` class after that change is merged, if it's a property of all templates.